src/main/java/ru/hh/jetty/HHServerConnector.java

```
@@ -0,0 +1,119 @@
+    super(server, acceptors, selectors, factories);
+  }
+
+  public HHServerConnector(@Name("server") Server server,
+                           @Name("factories") ConnectionFactory... factories) {
+    super(server, factories);
+  }
+
+  public HHServerConnector(@Name("server") Server server,
+                           @Name("sslContextFactory") SslContextFactory sslContextFactory) {
+    super(server, sslContextFactory);
+  }
+
+  public HHServerConnector(@Name("server") Server server,
+                           @Name("acceptors") int acceptors,
+                           @Name("selectors") int selectors,
+                           @Name("sslContextFactory") SslContextFactory sslContextFactory) {
+    super(server, acceptors, selectors, sslContextFactory);
+  }
+
+  public HHServerConnector(@Name("server") Server server,
+                           @Name("sslContextFactory") SslContextFactory sslContextFactory,
+                           @Name("factories") ConnectionFactory... factories) {
+    super(server, sslContextFactory, factories);
+  }
+
+  public HHServerConnector(@Name("server") Server server,
+                           @Name("executor") Executor executor,
+                           @Name("scheduler") Scheduler scheduler,
+                           @Name("bufferPool") ByteBufferPool bufferPool,
+                           @Name("acceptors") int acceptors,
+                           @Name("selectors") int selectors,
+                           @Name("factories") ConnectionFactory... factories) {
+    super(server, executor, scheduler, bufferPool, acceptors, selectors, factories);
+  }
+
+  @Override
+  protected SelectorManager newSelectorManager(Executor executor, Scheduler scheduler, int selectors) {
+    return new FailFastServerConnectorManager(executor, scheduler, selectors);
+  }
+
+  private class FailFastServerConnectorManager extends ServerConnectorManager {
```

И вот этот класс новый - для fail fast.